.github/workflows/security.yaml

```diff
@@ -0,0 +1,12 @@
+name: Security Audit
```

Comment:
I think that some of the things I did in [this repo](https://github.com/IronCoreLabs/unnamed-tenant-security-logging/blob/master/.github/workflows/security-audit.yaml) could be useful here as well.

Reply:
I think you're right. I'll copy all that over here.